### ISSUE TITLE
OpenStack changes needed for MOC multipath support

### ISSUE THREAD

**Issue by tzumainn** [CCI-MOC/esi#30]
From testing, there are some OpenStack bugs that prevent multipath support.

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1200974017553249/1200974083173684) by [Unito](https://www.unito.io)

**Comment by tzumainn** [comment]
https://review.opendev.org/c/openstack/ironic/+/775681

**Comment by tzumainn** [comment]
https://review.opendev.org/c/openstack/cinder/+/775706

**Comment by tzumainn** [comment]
Ironic change merged; still waiting on cinder

**Comment by tzumainn** [comment]
Cinder patch updated based on feedback and testing.

**Comment by tzumainn** [comment]
This is no longer needed; closing.
